PublicGitArchive/pga/cmd/get.go

```diff
@@ -46,7 +46,11 @@ Alternatively, a list of .siva filenames can be passed through standard input.`,
 			if err != nil {
 				return fmt.Errorf("could not read from standard input: %v", err)
 			}
-			filenames = strings.Split(string(b), "\n")
+			// to avoid empty strings
+			splitFn := func(c rune) bool {
+				return c == '\n'
+			}
+			filenames = strings.FieldsFunc(string(b), splitFn)
```

Comment:
I think it'd be easier to explicitly drop the empty lines inside of `downloadFilenames` and leave this code as is.

Basically start the for loop with:

```go
for _, filename := range filenames {
   filename = strings.TrimBlank(filename)
   if filename == "" {
      continue
   }
}
```